hue/hue/base/kustomization.yaml

```diff
@@ -0,0 +1,133 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - hive-site-xml-secret.yaml
+  - hue-db-init-job.yaml
+  - hue-dc.yaml
+  - hue-ini-secret.yaml
+  - hue-mysql-dc.yaml
+  - hue-mysql-pvc.yaml
+  - hue-mysql-secret.yaml
+  - hue-mysql-service.yaml
+  - hue-mysqld-exporter-secret.yaml
+  - hue-route.yaml
+  - hue-secret.yaml
+  - hue-service.yaml
+
+commonLabels:
+  opendatahub.io/component: "true"
+  component.opendatahub.io/name: hue
+  component.opendatahub.io/part-of: datacatalog
```

Comment:
I like this:)

Reply:
It kinda mimicks the https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ scheme :)

Reply:
Is this something we need to change? If not, can we mark as Resolved?

Reply:
I think it can stay. :wink: